astroplan/core.py

```diff
@@ -359,6 +415,22 @@ def altaz(self, time, target=None, obswl=None):
             If ``target`` is `None`, returns `~astropy.coordinates.AltAz` frame.
             If ``target`` is not `None`, returns the ``target`` transformed to
             the `~astropy.coordinates.AltAz` frame.
+
+        Examples
+        --------
+        Create an instance of the `~astropy.coordinates.AltAz` frame for an
+        observer at Apache Point Observatory at a particular time; and
+        transform that
```

Comment:
For this example I'd suggest to split it in two parts:

1. Up to the creation of the `altaz_frame`
2. Starting with the definition of `target` and transforming it.

The "and transform that" part here looks out of place.
If you split it in two sections you can make this a full sentence that starts the second part.